ui/dashboards/src/components/PanelDrawer/PanelDrawer.tsx

```diff
@@ -164,21 +184,12 @@ const PanelDrawer = () => {
             </FormControl>
           </Grid>
           <Grid item xs={8}>
-            <Stack spacing={2} sx={{ flexGrow: '1' }}>
-              <TextField
-                required
-                label="Panel Name"
-                value={panelName}
-                variant="outlined"
-                onChange={handlePanelNameChange}
-              />
-              <TextField
-                label="Description"
-                value={panelDescription}
-                variant="outlined"
-                onChange={handlePanelDescriptionChange}
-              />
-            </Stack>
+            <TextField
+              label="Panel Description"
+              value={panelDescription}
+              variant="outlined"
+              onChange={handlePanelDescriptionChange}
+            />
```

Comment:
Moved these `<input>`s around to match the Chronosphere design more

Reply:
Nice! In the future, we'll probably want to move some of these into @perses-dev/components so they're easier to share (this PR looks good, I'm thinking about more complicated controls in the future that diverge from MUI)

Reply:
Ya, like start having our own form field components in the components package?